frontend/portals/MaintenanceMode/index.jsx

```diff
@@ -1,6 +1,10 @@
 import appConfig from '@shopgate/pwa-common/helpers/config';
 import I18n from '@shopgate/pwa-common/components/I18n';
+import { Link } from '@shopgate/engage/components';
+import Button from '@shopgate/pwa-ui-shared/Button';
 import { getUserEmail } from '@shopgate/pwa-common/selectors/user';
+import { getClientInformation } from '@shopgate/engage/core';
+import { isIos } from '@shopgate/pwa-common/selectors/client';
```

Comment:
isIso import can also come from the engage core library as can openPageExtern so these can be added to the import of getClientInformation. 
`import { getClientInformation, isIos, openPageExtern } from '@shopgate/engage/core';`